.travis.yml

```diff
@@ -15,4 +14,4 @@ before_install:
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.9 100
 script:
   - mix format --check-formatted
-  - mix test
+  - mix test --include shm_tmpfs --include shm_resizable
```

Comment:
These tags are excluded only on platforms that do not have "/dev/shm" path, so this is not needed